scripts/sct_detect_pmj.py

```diff
@@ -17,62 +17,80 @@
-                                 ' with one voxel (value=50) located at the predicted PMJ position.\n'
-                                 ' If the PMJ is not detected, nothing is output.')
-    parser.add_option(name="-i",
-                        type_value="file",
-                        description="input image.",
-                        mandatory=True,
-                        example="t2.nii.gz")
-    parser.add_option(name="-c",
-                        type_value="multiple_choice",
-                        description="type of image contrast, if your contrast is not in the available options (t1, t2), use t1 (cord bright / CSF dark) or t2 (cord dark / CSF bright)",
-                        mandatory=True,
-                        example=["t1", "t2"])
-    parser.add_option(name="-s",
-                        type_value="file",
-                        description="SC segmentation or centerline mask. Provide this mask helps the detection of the PMJ by indicating the position of the SC in the Right-to-Left direction.",
-                        mandatory=False,
-                        example="t2_seg.nii.gz")
-    parser.add_option(name="-ofolder",
-                        type_value="folder_creation",
-                        description="Output folder",
-                        mandatory=False,
-                        example="My_Output_Folder/")
-    parser.add_option(name='-qc',
-                      type_value='folder_creation',
-                      description='The path where the quality control generated content will be saved',
-                      default_value=None)
-    parser.add_option(name="-igt",
-                      type_value="image_nifti",
-                      description="File name of ground-truth PMJ (single voxel).",
-                      mandatory=False)
-    parser.add_option(name="-r",
-                        type_value="multiple_choice",
-                        description="Remove temporary files.",
-                        mandatory=False,
-                        default_value="1",
-                        example=["0", "1"])
-    parser.add_option(name="-v",
-                        type_value='multiple_choice',
-                        description="Verbose: 0 = nothing, 1 = classic, 2 = expended",
-                        mandatory=False,
-                        example=["0", "1", "2"],
-                        default_value="1")
+
+    parser = argparse.ArgumentParser(
+        description='Detection of the Ponto-Medullary Junction (PMJ).\n'
+                    ' This method is machine-learning based and adapted for T1w-like or T2w-like images.\n'
+                    ' If the PMJ is detected from the input image, a nifti mask is output ("*_pmj.nii.gz")\n'
+                    ' with one voxel (value=50) located at the predicted PMJ position.\n'
+                    ' If the PMJ is not detected, nothing is output.',
+        add_help = None,
```

Comment:
please check your space (PEP8). Fix everywhere

Reply:
Sounds good! Changes made in following functions:

sct_denoising_onlm: Commit [3fc757f](https://github.com/neuropoly/spinalcordtoolbox/commit/3fc757f2b6db8c36041778128caca8048496fcbc)
sct_deepseg_gm: Commit [97c7ec9](https://github.com/neuropoly/spinalcordtoolbox/commit/97c7ec91343a38607322db0290adfe5f70e15014)
sct_deepseg_lesion: Commit [92383f5](https://github.com/neuropoly/spinalcordtoolbox/commit/92383f551cb45a93227096f66962a0c7feca0994)
sct_deepseg_sc: Commit [28e695e](https://github.com/neuropoly/spinalcordtoolbox/commit/28e695e545b69cd1a3c8e788f438c92d8df813e3)
sct_detect_pmj: Commit [1051920](https://github.com/neuropoly/spinalcordtoolbox/commit/105192069d0f117e8881a3d40de342f83756057c)
sct_dice_coefficient: Commit [751e4f8](https://github.com/neuropoly/spinalcordtoolbox/commit/751e4f880fc18605268bc6715b01f7871f30e79c)
sct_dmri_compute_bvalue: Commit [38e697a](https://github.com/neuropoly/spinalcordtoolbox/commit/38e697a77686123ff4323fa8ddd4b431f438bf4f)
sct_dmri_compute_dti: Commit [d36f14e](https://github.com/neuropoly/spinalcordtoolbox/commit/d36f14e9e83b5210f5bcc3b422a29901b85a99ae)
sct_dmri_concat_bvals: Commit [db56c17](https://github.com/neuropoly/spinalcordtoolbox/commit/db56c174d23a866081e25940c2d5f91a2afb5c7c)
sct_dmri_concat_bvecs: Commit [9ad916b](https://github.com/neuropoly/spinalcordtoolbox/commit/9ad916b191af016d4b61b4eae840e6fa57555668)